packages/pn-personafisica-webapp/src/api/interceptors.ts

```diff
@@ -1,8 +1,23 @@
-import { EnhancedStore } from "@reduxjs/toolkit";
-import { apiClient } from "./apiClients";
+import { EnhancedStore } from '@reduxjs/toolkit';
+import { NotificationDetail, NotificationStatus, TimelineCategory } from '@pagopa-pn/pn-commons';
+import { apiClient } from './apiClients';
+
+// eslint-disable-next-line functional/no-let
+let axiosResponseInterceptor: number; // outer variable
```

Comment:
merge the feat/cancel-notification and remove the clearInterceptor usages, and the variable axiosRequestInterceptor and axiosResponseInterceptor

Reply:
do it